package.json

```diff
@@ -17,8 +17,8 @@
     "url": "https://github.com/Automattic/sensei/issues"
   },
   "engines": {
-    "npm": "^9.3.1 || ^10",
-    "node": "^18.14.0"
+    "npm": "^10.2.3",
+    "node": ">=20.10.0"
```

Comment:
Is there a reason to keep it v20 here and not switch to v22 as in `.nvmrc`? I think it would be best to use the same version in all places to avoid environment issues.

Reply:
I've followed the instructions from the task: https://github.com/Automattic/sensei/issues/7722

Reply:
Oh, ok, makes sense to follow Guttenberg. 👍 I was a bit confused about `.nvmrc` being a higher version, but I guess that's fine.

Reply:
No problem. Thanks for testing it.

If everything is fine, can you approve it?